app/views/groups/show.html.erb

```diff
@@ -0,0 +1,9 @@
+<p style="color: green"><%= notice %></p>
+<br/>
+<%= render @group %>
+
+<div class="footer-buttons">
+  <%= link_to "Back to groups", groups_path, class:"inner-buttons" %>
+
+  <%= button_to "Destroy this group", @group, method: :delete, class:"new-buttons" %>
+</div>

```

Comment:
- [ ] As suggested by the previous reviewer, please ensure that this page shows a list of transactions and also a button to add more transactions at the bottom of the page 🙏

Reply:
Fixed.